docs/boards/frdm_k64f.md

```diff
@@ -0,0 +1,85 @@
+ZJS Board Support for FRDM-K64F
+===============================
+
+* [Introduction](#introduction)
+* [Available Pins](#available-pins)
+* [Sample Apps](#sample-apps)
+
+Introduction
+------------
+The FRDM-K64F is pretty well supported by ZJS. This page lists the available
+pins and how to refer to them in ZJS.
+
+Available Pins
+--------------
+It will be helpful to refer to the two diagrams
+[here](https://os.mbed.com/platforms/FRDM-K64F/#component-pinout)
+when working with the K64F.
+
+### GPIO Pins
+The FRDM-K64F has 16 general purpose I/O pins, D0 - D15.
+
+If you look at the diagrams above, the pin D0 in the bottom right corner is
+also labeled PTC16. This means it's pin 16 on the 'C' GPIO controller. The full
+pin specification for that pin is "GPIO_2.16" (2 because the controllers A-E are
+0-indexed, 0 to 4). All the other pins listed with a blue box of the format
+PTC16 can also be used as GPIOs. For example, the pins that can also be used as
+analog inputs (A0 - A5) also have GPIO names. (These work as GPIOs unless you
```

Comment:
Should we consider adding a DBG_PRINT for these kind of multi use pin cases?  So the user could get some feedback on why the pin wasn't behaving as expected?  Doesn't have to be done in this patch, but might be something to consider for the future.

Reply:
I guess you mean if it's configured as an AIO but they try to use it as a GPIO... Yeah, I guess that would be possible but we'd want to be careful to not increase code size for release case. So this is not too bad right now because it will fail and tell them "invalid pin" which might bring them to the docs to investigate. I'll add it to my infinite TODO list. :)